### PR TITLE
Fix mysql/maria sample to have a sampleapp and data

### PR DESCRIPTION
**Pull request by weshayutin**
This should fix issue:
https://github.com/openshift/oadp-operator/issues/609

This adds a mediawiki app that uses mariadb and has
data associated with it.

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

**Comment by kaovilai** [comment]
I love your dynamic on this issue.

File: tests/e2e/sample-applications/mysql-persistent/mysql-persistent-template.yaml [hunk 1]
```diff
@@ -45,92 +56,163 @@ items:
...
+              - name: MEDIAWIKI_SITE_NAME
+                value: Mediawiki
+              - name: MEDIAWIKI_SITE_LANG
+                value: en
+              - name: MEDIAWIKI_ADMIN_USER
+                value: admin
+              - name: MEDIAWIKI_ADMIN_PASS
+                value: changeme
+              - name: MEDIAWIKI_SITE_SERVER
+                value: mediawiki-mariadb-persistent.apps.cluster-wdh0321dpk48.wdh0321dpk48.mg.dog8code.com
```
Comment (kaovilai) [inline tests/e2e/sample-applications/mysql-persistent/mysql-persistent-template.yaml:178]:
```suggestion
                value: mediawiki.mariadb-persistent.svc
```

**Review by kaovilai — APPROVED** [review]

**Comment by weshayutin** [comment]
ok.. think we're gtg https://prow.ci.openshift.org/pr-history/?org=openshift&repo=oadp-operator&pr=612  on https://github.com/openshift/oadp-operator/pull/612 w/ two follow up items.    https://github.com/openshift/oadp-operator/issues/619 and https://github.com/openshift/oadp-operator/issues/622

**Pull request merged**